static/grapheneos.css

```diff
@@ -52,31 +52,42 @@ nav ul {
 }
 
 nav {
-    background-color: #212121; /* grey900 */
-    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 5px 0 rgba(0, 0, 0, 0.12), 0 3px 1px -2px rgba(0, 0, 0, 0.2);
+    background-color: #c2185b;
+    -webkit-box-shadow: 0px 1px 5px 0px rgba(0,0,0,0.75);
```

Comment:
Should avoid prefixes when they aren't required for current generation browsers.